docs/matrix-creator/device-setup.md

```diff
@@ -0,0 +1,23 @@
+## Hardware Prerequisites
```

Comment:
Should we recommend to use pi3 or above? I'm not sure if all of our stuff still work on pi 1(need to test)